habitat_sim/agent/controls/pyrobot_noisy_controls.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import attr
+import magnum as mn
+import numpy as np
+
+import habitat_sim.bindings as hsim
+from habitat_sim import utils
+
+from .controls import ActuationSpec, SceneNodeControl, register_move_fn
```

Comment:
NIT: if possible we should move to absolute imports instead of relative imports.